### DISCUSSION TITLE
hmm, can't get it to connect

### DISCUSSION THREAD

**Discussion by scottcrevier** [ecdye/TrimPy#15] [General]
I just fired this up for the first time. Running it from my Windows 10 computer. Here's my command:

`python -m TrimPy --ip=192.168.XXX.XXX --query-pattern=1`

with the correct IP address, and there's the output:

`ConnectionRefusedError: [WinError 10061] No connection could be made because the target machine actively refused it`

I'm brand new to using Trimlight (though not new to programming, APIs, etc). Is there something I have to do to the system to allow it to talk to something like this?

**Comment by ecdye** [comment]
Not that I am aware of. Are you using a Trimlight Select system?

**Reply by scottcrevier** [reply]
Well, I don't know. How can I tell if it's "Trimlight Select"? The Android app that I use is "Trimlight Edge" v2.5.3.

**Reply by ecdye** [reply]
That means that you have an Edge system, which is the newer version of the product. This program will not work with that

**Reply by scottcrevier** [reply]
OK, the installer told me that I could do so. Thanks.
